setup.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import codecs
+from setuptools import setup
+
+
+def read(fname):
+    file_path = os.path.join(os.path.dirname(__file__), fname)
+    return codecs.open(file_path, encoding="utf-8").read()
+
+
+setup(
+    name="pytest-kiwitcms",
+    version="0.1.0",
+    author="Dmitry Dygalo",
+    author_email="dadygalo@gmail.com",
+    maintainer="Dmitry Dygalo",
+    maintainer_email="dadygalo@gmail.com",
+    license="GNU GPL v3.0",
```

Comment:
`GPLv3+` is what we have in other setup.py files. I'm not quite sure which format is the correct one.